src/strands/agent/agent.py

```diff
@@ -210,6 +213,7 @@ def __init__(
         messages: Optional[Messages] = None,
         tools: Optional[list[Union[str, dict[str, str], Any]]] = None,
         system_prompt: Optional[str] = None,
+        structured_output_type: Optional[Type[BaseModel]] = None,
```

Comment:
```suggestion
        structured_output_type: Type[BaseModel] | None = None,
```

Reply:
agreed. The team thought it should `structured_output_type` but as discussed that sounds overly verbose. Are you aligned with `output_schema`? @zastrowm @Unshure